packages/expansion-hub/src/open-rev-hub.ts

```diff
@@ -54,6 +54,21 @@ async function getSerialPortPathForExHubSerial(
     throw new Error(`Unable to find serial port for ${serialNumber}`);
 }
 
+/**
+ * Closes the given Serial port and removes it from the open serial ports
+ * list. This should be the preferred way to close a Serial port.
+ *
+ * @param serial the Serial port to close
+ */
+export function closeSerial(serial: Serial) {
+    for(let [path, port] of openSerialMap.entries()) {
```

Comment:
If we exposed the serialPortPath for Serial objects, we could avod this lookup. It's extremely unlikely for there to be a significant number of USB-connected RevHubs, though, so maybe this loop doesn't matter.